devices/gd32e230.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The gd32-rs authors.
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+_svd: ../svd/gd32e230.svd
+
+_modify:
+  name: "GD32E230"
```

Comment:
Do you need this? I did it for the GD32F1x0 devices because they are all based on the same SVD file, but seeing as these ones have separate SVD files already I wouldn't expect it to be necessary.